VCFv4.3.tex

```diff
@@ -580,7 +602,13 @@ \section{INFO keys used for structural variants}
 ##INFO=<ID=SVLEN,Number=.,Type=Integer,Description="Difference in length between REF and ALT alleles">
 \end{verbatim}
 \normalsize
-One value for each ALT allele. Longer ALT alleles (e.g.\ insertions) have positive values, shorter ALT alleles (e.g.\ deletions) have negative values.
+SVLEN indicates the relative size of the ALT allele compared to the REF allele.\
+If ALT is longer than REF (e.g. insertions), SVLEN must be positive; if ALT is shorter than REF (e.g. deletions), SVLEN must be negative.
```

Comment:
the backlashes are needed to avoid a wider whitespace (I think it's interpreted as an end-sentence period)
```suggestion
If ALT is longer than REF (e.g.\ insertions), SVLEN must be positive; if ALT is shorter than REF (e.g.\ deletions), SVLEN must be negative.
```